CHANGES.md

```diff
@@ -2,6 +2,14 @@
 
 ### 1.99 - 2022-11-01
 
+#### Major Announcements :loudspeaker:
+
+- CesiumJS now uses npm workspaces:
```

Comment:
workspaces is a bit of an implementation detail. For an end user, what matters here are the new packages. I would probably lead with the two new npm packages, and follow with the workspace explanation.

Reply:
Also the release zip grew in size quite a bit. Is it possible to reduce some of the redundancy? Via the release zip, it's probable that user won't be using `packages/` directly.

Reply:
- Updated `CHANGES.md`.
- Removed the `packages/*/Build/Specs` as we did not originally publish the ZIP file with those.
- Removed sourcemaps from bundles as we did not originally publish the ZIP file with those.